cranelift/filetests/filetests/runtests/srem.clif

```diff
@@ -1,13 +1,11 @@
 test interpret
 test run
-target aarch64
-target s390x
-target x86_64
```

Comment:
Note, these are not duplicates of the ones below!

These targets, run this test file without the `avoid_div_traps` flag, and the ones below run the test file with the `avoid_div_traps` flag. Although we should probably clarify that in a comment

Reply:
ok, why I didn't see any test want to div by zero?

Reply:
what will cranelift expect to produce if `100/0` ??

Reply:
@yuyang-ok we get divide-by-zero coverage via the Wasm tests run by Wasmtime.

Reply:
@cfallin  yes,But I think wasmtime test will all want to cause trap.
I see test in `urem.clif` set this flag.
~~~
function %urem_i64(i64, i64) -> i64 {
block0(v0: i64,v1: i64):
    v2 = urem v0, v1
    return v2
}
; run: %urem_i64(0, 1) == 0
; run: %urem_i64(2, 2) == 0
; run: %urem_i64(1, -1) == 1
; run: %urem_i64(3, 2) == 1
; run: %urem_i64(19, 7) == 5
; run: %urem_i64(3, -2) == 3
; run: %urem_i64(-19, 7) == 4
; run: %urem_i64(-57, -5) == -57
; run: %urem_i64(0, 104857600000) == 0
; run: %urem_i64(104857600000, 511) == 398
; run: %urem_i64(0xC0FFEEEE_DECAFFFF, 8) == 7
; run: %urem_i64(0xC0FFEEEE_DECAFFFF, -8) == 0xC0FFEEEE_DECAFFFF
; run: %urem_i64(0x80000000_00000000, -2) == 0x80000000_00000000
~~~
I didn't see you want to divide by zero.

Reply:
Yeah, We can't test traps via filetests yet! But it was something that was brought up recently in https://github.com/bytecodealliance/wasmtime/issues/4781.

Adding div by zero tests would be nice, but if we can't catch the traps properly we can't include it in the test suite (since it makes it crash)

Reply:
Yeah, We can't test traps via filetests yet! But it was something that was brought up recently in #4781.

Adding div by zero tests would be nice, but if we can't catch the traps properly, we can't include it in the test suite (since it makes it crash).

Reply:
look like we can test divide by zero in `urem.clif` because we can `avoid_div_traps` , but I didn't know whick value should produce.